README.md

```diff
@@ -81,6 +87,7 @@ require "bump/tasks"
 ```
 
     rake bump:current                           # display current version
+    rake bump:show-next INCREMENT=minor         # display next minor version
```

Comment:
traditionally rake tasks use underscores, prefer to stick with that even if it's not consistent

Reply:
yeah yeah I know but I preferred to follow the same name as we have for bump script